regress/core/out_marc21_expected

```diff
@@ -0,0 +1,810 @@
+31|<record xmlns="http://www.loc.gov/MARC21/slim"><datafield tag="034" ind1="1" ind2=" "><subfield code="a">a</subfield><subfield code="d">W0044903</subfield><subfield code="e">W0044903</subfield><subfield code="f">N0540319</subfield><subfield code="g">N0540319</subfield></datafield><datafield tag="034" ind1="1" ind2=" "><subfield code="a">a</subfield><subfield code="d">W0044940</subfield><subfield code="e">W0044940</subfield><subfield code="f">N0540313</subfield><subfield code="g">N0540313</subfield></datafield><datafield tag="034" ind1="1" ind2=" "><subfield code="a">a</subfield><subfield code="d">W0044944</subfield><subfield code="e">W0044944</subfield><subfield code="f">N0540311</subfield><subfield code="g">N0540311</subfield></datafield><datafield tag="034" ind1="1" ind2=" "><subfield code="a">a</subfield><subfield code="d">W0044943</subfield><subfield code="e">W0044943</subfield><subfield code="f">N0540308</subfield><subfield code="g">N0540308</subfield></datafield></record>
+32|<record xmlns="http://www.loc.gov/MARC21/slim"><datafield tag="034" ind1="1" ind2=" "><subfield code="a">a</subfield><subfield code="d">W0043259</subfield><subfield code="e">W0043137</subfield><subfield code="f">N0540941</subfield><subfield code="g">N0540847</subfield></datafield><datafield tag="034" ind1="1" ind2=" "><subfield code="a">a</subfield><subfield code="d">W0043445</subfield><subfield code="e">W0043248</subfield><subfield code="f">N0541149</subfield><subfield code="g">N0541054</subfield></datafield></record>
+34|<record xmlns="http://www.loc.gov/MARC21/slim"><datafield tag="034" ind1="1" ind2=" "><subfield code="a">a</subfield><subfield code="d">W0043356</subfield><subfield code="e">W0043356</subfield><subfield code="f">N0541105</subfield><subfield code="g">N0541105</subfield></datafield></record>
+35|<record xmlns="http://www.loc.gov/MARC21/slim"><datafield tag="034" ind1="1" ind2=" "><subfield code="a">a</subfield><subfield code="d">W0043704</subfield><subfield code="e">W0043536</subfield><subfield code="f">N0541104</subfield><subfield code="g">N0541001</subfield></datafield></record>
+36|<record xmlns="http://www.loc.gov/MARC21/slim"><datafield tag="034" ind1="1" ind2=" "><subfield code="a">a</subfield><subfield code="d">W0044148</subfield><subfield code="e">W0042904</subfield><subfield code="f">N0541325</subfield><subfield code="g">N0540455</subfield></datafield></record>
+hdddmmss|1|010400000004000000010100000085EB51B81E4513C024E09B5713074B400101000000FAA44FFAA44F13C0547698BADC064B400101000000A62EB73FC85013C0B9FD4186CA064B4001010000003B4C5D6E7F5013C0D148C037AF064B40
```

Comment:
Comparing rounded WKT rather than WKB will make this more robust over small system differences.